### ISSUE TITLE
Artifacts' json doesn't recognise nbt correctly

### ISSUE THREAD

**Issue by ztblat** [phantamanta44/tinkers-evolution#145]
{
    "free_mods": 0,
    "data_tag":{
      "Quality":{
        "Name":"quality.loser.name"
      },
      "Stats":{
        "Attack":255.0
      }
}
  }
255.0 is not recognised as a float.
Free_mods set to 0, there are still 2 free_mods in the game
and will not be inherited by toolforge nbt.

**Comment by phantamanta44** [comment]
Yeah, the NBT parser is quite bad and currently does not support floats at all. As for the modifier thing, the `free_mods` field is just for adding *additional* free modifier slots; the baseline is still set to the default 3 slots (or whatever you might have changed it to with Tinkers' Tool Leveling)
